packages/aws-cdk/lib/init.ts

```diff
@@ -156,6 +162,22 @@ export class InitTemplate {
     }
   }
 
+  private needsShell(language: string, type: string): boolean {
+    return (language === 'csharp' || language === 'fsharp') && (type === 'app' || type === 'default');
+  }
+
+  private async copyShellTo(targetDirectory: string): Promise<string> {
```

Comment:
Same here. What if we add files or anything, or those files start taking additional dependencies?

I wonder if we just change the logic a little, executing the `add-project.hook.js` file in its *original* location with `cwd` set to the target directory won't just do it, and be a lot easier?

Reply:
This is better and now I'm mad that I didn't think of it.